lib/matplotlib/gridspec.py

```diff
@@ -232,38 +232,42 @@ def _normalize(key, size, axis):  # Includes last index.
 
 class GridSpec(GridSpecBase):
     """
-    Specifies the geometry of the grid that a subplot can be placed in.
+    A grid layout to place subplots within a figure.
 
-    The location of grid is determined by similar way as the SubplotParams.
+    The location of the grid cells is determined in a similar way as
+    `~.figure.SubplotParams` using *left*, *right*, *top*, *bottom*, *wspace*
```

Comment:
would just edit this myself, but github isn't letting me. I think it should be `to` instead of `as`